stories/sample-data/sample5.json

```diff
@@ -0,0 +1,106 @@
+{
+  "id": 1815,
+  "aws_id": "df-085535826XRNXUA5LNGN",
+  "job_name": "clra8067dk191127_xnzh4lp8_GnO_201912141049",
```

Comment:
we may want to redact some details like these (and preferably hide brand details like `bell`)